adobe/aem-project-archetype
# Enhancement - Deprecate TSLint since it is officially deprecated in favor of ESLin…

Deprecated TSLint since it is _officially deprecated_ in favor of ESLint with Typescript rules



Since TSLint has been deprecated. We need to update the linter to use ESlint since it now has official TSLint rules. It is also the recommend linter going forward for Typescript projects.

TESTING STEPS:
1. After running `npm install` in UI.Frontend module, run `npm run dev`.
2. There should be one warning that will output. It is related to the unused var assigned in `HelloWorld.ts`.

This shows the new linter is set up and configured correctly. 



## Related Issue






## Motivation and Context



This fixes the inevitable problem of TSLint being deprecated and, as such, no longer receiving active support. It also aligns our linter with the new standard, which is ESLint with Typescript definitions

## How Has This Been Tested?

Yes, see above steps for testing. This can be tested with either the `dev` or `prod` flag.





## Screenshots (if appropriate):

## Types of changes

Deprecation replacement.




## Checklist:
